examples/nucleo_f429zi/rtc_mcp7941x/project.xml

```diff
@@ -0,0 +1,14 @@
+<library>
+  <extends>modm:nucleo-f429zi</extends>
+  <options>
+    <option name="modm:build:build.path">../../../build/nucleo_f429zi/blink</option>
```

Comment:
Fail. I'll push a fix for this later.

Why do we specify build paths explicitly for every example?